examples/transport-with-transfer.yaml

```diff
@@ -0,0 +1,169 @@
+    receiver: https://claudia.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+    transferPossession: true
+
+  - '@id': claudia:7dd207e3-8f11-4fe3-95f8-d02f6c180be3
+    '@type': EconomicEvent
+    outputOf: alice:533f6e56-6c7d-4a5b-b9c9-1a8adafd8969
+    action: take
+    resourceInventoriedAs: claudia:27be5cab-d740-4194-9298-1661a69d9d95 # claudia's inventory / bill of lading on the truck
+    provider: https://alice.example/
+    receiver: https://claudia.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+    transferPossession: true
+
+  # claudia transports the apples
+
+  - '@id': claudia:633f6e56-6c7d-4a5b-b9c9-1a8adafd8960
+    '@type': Process
+    skos:note: Claudia transports the apples
+
+  - '@id': claudia:fd399b37-0740-4a68-a184-1e655021ca21
+    '@type': EconomicEvent
+    inputOf: claudia:633f6e56-6c7d-4a5b-b9c9-1a8adafd8960
+    action: load
+    resourceInventoriedAs: claudia:27be5cab-d740-4194-9298-1661a69d9d95 
+    provider: https://claudia.example/
+    receiver: https://claudia.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+
+  - '@id': claudia:57f1c1d0-432e-4bfa-9d32-002b8955a708
+    '@type': EconomicEvent
+    outputOf: claudia:633f6e56-6c7d-4a5b-b9c9-1a8adafd8960
+    action: unload
+    resourceInventoriedAs: claudia:27be5cab-d740-4194-9298-1661a69d9d95
+    provider: https://claudia.example/
+    receiver: https://claudia.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+
+  # bob receives the apples
+
+  - '@id': bob:ee652454-ac08-4f5d-ae84-6b1753887086
+    '@type': Process
+
+  - '@id': bob:7a63ea10-b1c3-441a-9a08-fb8630c02614
+    '@type': EconomicEvent
+    inputOf: bob:ee652454-ac08-4f5d-ae84-6b1753887086
+    action: give
+    resourceInventoriedAs: claudia:27be5cab-d740-4194-9298-1661a69d9d95 
+    provider: https://claudia.example/
+    receiver: https://bob.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+    transferPossession: true
+
+  - '@id': bob:8e5fe80d-a769-4bd5-89e5-2136d33eab9f
+    '@type': EconomicEvent
+    outputOf: bob:ee652454-ac08-4f5d-ae84-6b1753887086
+    action: take
+    resourceInventoriedAs: bob:9bd19194-a36d-4a1f-896b-8082887962cb 
+    provider: https://claudia.example/
+    receiver: https://bob.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
+    transferPossession: true
+
+  # receipt of the apples triggers finalizing the transfer of ownership
+
+  - '@id': alice:ad56a7ed-be3c-4937-a3fb-0f156bcd2c47
+    '@type': EconomicEvent
+    inputOf: alice:33e8933b-ff73-4a01-964a-ca7a98893083
+    action: give
+    resourceInventoriedAs: alice:21f361a6-2375-46bb-b192-c21b5ba833bf
+    provider: https://alice.example/
+    receiver: https://bob.example/
+    quantifiedAs:
+      qudt:unit: unit:Kilogram
+      qudt:numericValue: 30
```

Comment:
So, alice gives possession to claudia, claudia gives possession to bob.  I was thinking possession doesn't decrement/increment.  Then at the end, alice give ownership to bob, that decements and increments.

But that is somewhat confusing, I agree.  Alice needs to know she has the apples in her inventory (in the sense of ownership, part of her "assets", but also something she is responsible for still, in case of problems with shipping for example); but that they are not available because she has given possession of them.  So that is the "inventory in transit".  Available quantity could have its own attribute, currentAvailableQuantity or something.  Or it could be calculated for whatever time period someone needs to know, based on various commitments, transfer of possession, etc.

Reply:
I still think we may go to far from having a clear notion of physical observable resource at some location (or in transit) with certain observable `currentQuantity`. Probably we may need to solve first situation in #365 where we have one physical resource where different agents own different quantities of it. I think such physical resource could stay located in some place or stay on vehicle 'in-transit' and still processes that involve it shouldn't care who owns how much of it.

Reply:
Just to note another complication, an agent could have an inventoried resource that doesn't say its location because it might be in several places, but is identified with the agent and the resource specification/classification, not location.  That would have a currentQuantity.

But yes, the currentQuantity for sure needs to be clarified - there is clear observable quantity, there is quantity that is accounted for on the books, .....

Reply:
I think this again shows weak notion of actual physical resource and more focus on accounting records. Physical resource can't stay in multiple places at the same time, most likely each physical resource needed to get transported to one's current location. At the same time 'accounting records' or ownership rights don't seem to care that much about physical location of the resource.

Reply:
It is possibly more focused on accounting needs, but also just assumes people working in the operational side already know where things are located and don't need a computerized record at that level of detail.  It is a valid scenario operationally for there to be one recorded resource per resource specification even if there are separate piles or whatever in different locations, thinking of a space where no transportation is required and you can pretty much see everything easily.  So that applies to one agent scope.

On the topic more broadly, One thing I've discovered from this whole transfer exercise is that different agents may need different records of the same physical resource.  Like the "owner" and the "shipper".  So in different agent scopes, yes I'm seeing multiple records of one resource, some of them more operational, some of them more accounting or responsibility based.